dashboard/app/views/lti/v1/integration_status.html.haml

```diff
@@ -8,16 +8,18 @@
               %h1= t('lti.integration.success_header')
           .row
             .span10
-              %p= t('lti.integration.success_body').html_safe
-              %p= t('lti.integration.guide_instructions').html_safe
+              %p= t('lti.integration.success_body')
+              %p
+                != I18n.t('lti.integration.guide_instructions', lms_guides: 'https://support.code.org/hc/en-us/articles/23621907533965-Install-Code-org-Integrations-for-your-Learning-Management-System', contact_us: 'https://code.org/contact', markdown: true)
```

Comment:
For the reused links like `contact_us`, you could put them into [shared_constants](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/shared_constants.rb#L624-L632). For the CPA parent mailer we had a bunch of reused links that actually had to be changed, and updating them all individually can get messy.

Reply:
Good call, I moved these to shared_constants.